test/fixtures/users.yml

```diff
@@ -1,31 +1,891 @@
-ohm:
+users_001:
   name: Georg Ohm
   email: gohm@example.com
   password_digest: <%= User.digest('password') %>
   admin: true
 
-watt:
+users_002:
   name: James Watt
   email: jwatt@example.com
   password_digest: <%= User.digest('password') %>
 
-pasteur:
+users_003:
   name: Louis Pasteur
   email: lpasteur@example.com
   password_digest: <%= User.digest('password') %>
 
-palmer:
+users_004:
   name: John Palmer
   email: jpalmer@example.com
   password_digest: <%= User.digest('password') %>
 
-tesla:
-  name: Nikola Tesla
-  email: ntesla@example.com
-  password_digest: <%= User.digest('password') %>
+users_005:
```

Comment:
See if you can get away without adding all these guys now that you know which tests work/don't work. We shouldn't be adding more fixtures than we need.